### ISSUE TITLE
Doesn't work after suspend/resume

### ISSUE THREAD

**Issue by eighteentee** [ModernFlyouts-Community/ModernFlyouts#729]
**Describe the bug:**

Does not work / display after suspend / resume 

**To Reproduce:**

1. Close laptop
2. Let it suspend
3. Open laptop and let it get to the desktop
4. ModernFlyouts not working - needs to be re-loaded

**Screenshots:**

**OS Version:**
 - Windows 10 Version: 21H1

**ModernFlyouts Version:**
 - Version: 0.9.3.0

**Additional context:**
ModernFlyouts needs to be reloaded every time the laptop is suspended or closed, then resumed.

**Comment by m4l0n** [comment]
It happens after I restart my Windows too

**Comment by JohnLGalt** [comment]
Question to both of you @eighteentee & @m4l0n  - are you using the executable DL from here(GitHub) or the Windows Store version?

I'm using WS ver and none of that happens.  So just curious if there may be a difference in execution and 'sty resident' with Store app.  It's also ver 0.9.3.0

**Comment by Samuel12321** [comment]
Good question @JohnLGalt, but the two are identical, in fact the msix published to GitHub is downloaded from the store so that it is signed by Microsoft.

**Comment by JohnLGalt** [comment]
Agreed.  However, if the bundle is not installed from the store, maybe some obscure setting in Windows is causing it to stop working?

**Comment by m4l0n** [comment]
@JohnLGalt I installed mine from WS.

**Comment by aronsky** [comment]
Any news about this? Any way to check the logs to help identify the issue? v0.9.3, from the Store, on Windows 11. Looks like it's crashing silently on resume from sleep (as the icon in the system tray is there, but disappears when I move the mouse over it).

**Comment by Shoggomo** [comment]
Until this is fixed, a possible workaround is to start the app on wakeup. This can be done over a scheduled task that runs the following command: `explorer.exe shell:appsFolder\32669SamG.ModernFlyouts_pcy8vm99wrpcg!App`.

Running the command works, but I haven't tested if the task works yet.
